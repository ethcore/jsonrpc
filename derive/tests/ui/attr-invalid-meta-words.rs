extern crate serde;
extern crate jsonrpc_core;
#[macro_use]
extern crate jsonrpc_derive;

use jsonrpc_core::Result;

#[rpc]
pub trait Rpc {
	/// Returns a protocol version
	#[rpc(name = "protocolVersion", Xmeta)]
	fn protocol_version(&self) -> Result<String>;
}

fn main() {}
