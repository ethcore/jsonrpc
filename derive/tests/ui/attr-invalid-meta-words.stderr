error: Invalid attribute parameter(s): 'Xmeta'. Expected 'meta, raw_params'
  --> $DIR/attr-invalid-meta-words.rs:8:2
   |
8  |       /// Returns a protocol version
   |  _____^
9  | |     #[rpc(name = "protocolVersion", Xmeta)]
10 | |     fn protocol_version(&self) -> Result<String>;
   | |_________________________________________________^
